dallinger/default_configs/global_config_defaults.txt

```diff
@@ -23,7 +23,6 @@ enable_global_experiment_registry = False
 language = en
 lock_table_when_creating_participant = True
 mode = debug
-publish_experiment = True
```

Comment:
Why not keep it like `publish_experiment = False` ?

Reply:
I any case _docs/source/configuration.rst_ needs to be adapted accordingly.

Reply:
If we set the default to `False` that means that MTurk deployment will fail by default, which I think is undesirable. Better to leave it unset I think.

Reply:
Docs updated

Reply:
But then, doesn't it fail because `Recruiter.validate_config` will be called anyway?

Reply:
Yes, good point. We will have to update `validate_config` so that it doesn't throw an error when the config variable is missing.

Unfortunately I am quite busy in the next few days and I appreciate this is blocking your release. I think it is best if you take over this PR and I can review any further changes you make.

Reply:
@pmcharrison Made a small change which should resolve the issue. Please have a look!